### DISCUSSION TITLE
vue3 and inertia

### DISCUSSION THREAD

**Discussion by Ricardosgeral** [spatie/laravel-permission#2281] [Q&A]
Hi 

Does spatie works out of the box with inertia and vue3?

What are the substitutes of directives like @can ... used in the blade directives?
Examples?

Best regards

**Comment by parallels999** [comment]
> What are the substitutes of directives like `@can` ... used in the blade directives?

`@can` comes from laravel's gate, that question must be on laravel framework, not here
